samples/WorkerBindingSamples/Program.cs

```diff
@@ -9,6 +10,8 @@ public class Program
     {
         public static void Main()
         {
+            Console.WriteLine($"Azure Functions .NET Worker (PID: { Environment.ProcessId }) initialized in debug mode. Waiting for debugger to attach...");
```

Comment:
Is this necessary? Technically it isn't always debug mode.

Reply:
Nope, this is just for me whilst I'm still testing E2E - will remove before merge